### PR TITLE
Docs for customizing drill-through

### PR DESCRIPTION
**Pull request by mazameli**
Feels a bit verbose, so gonna do another pass to break it up a bit and see if I can cut some verbiage.

### CODE REVIEW

File: docs/enterprise-guide/customizing-drill-through.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+This comes in handy when linking to a dashboard or question with multiple filters: your link template could for example reference two values for two filters in the dashboard you're linking to, like this so, `http://YOUR-METABASE-LOCATION.com/dashboard/5?name={{NAME}}&id={{ID}}`, where `name` and `id` are the filters. Note the ampersand `&` between the two filters in the link; you can string together any number of filters in your link templates by putting an ampersand between them.
+
+**Customizing link text**
+
+Additionally, you can optionally customize what text should be displayed in each cell of this column. You can also use the double braces variable syntax here. The way we've used this in the example above is to refer to the value of our `Product ID` column in a sentence that makes it clearer what will happen when the cell is clicked. So `Look up {{PRODUCT_ID}} in our catalog` gets turned into `Look up Awesome Concrete Shoes in our catalog` in the corresponding cell.
```
Comment (kdoh) [inline docs/enterprise-guide/customizing-drill-through.md]: `Additionally, you can optionally` stood out as a bit awkward to me.

File: docs/enterprise-guide/customizing-drill-through.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+
+### Customizing drill-through for a column in a table
+
+Customizing drill-through for a table is very similar to doing it for a chart. Usually what we’re trying to do with a table though is to make it so that clicking on a cell in a specific column will go to a custom destination. E.g., I might have a `Product ID` column, and I’d like to make it so that clicking on a specific product's ID will open up my product catalog web app to the page that has details for the product I clicked on.
+
+Again, in our example our table visualization is in a dashboard, so we’ll start by entering edit mode for our dashboard and then clicking the settings gear button on our table. If you were doing this on a standalone question, you’d click the gear button next to the visualization dropdown.
+
+From the `Columns` tab I'll find my `Product ID` column and click the gear next to it.
```
Comment (kdoh) [inline docs/enterprise-guide/customizing-drill-through.md]: The switch from `we` to `I` stood out here and the other instance below.

**Comment by mazameli** [comment]
This PR now additionally moves docs for enterprise features into a new enterprise edition guide (and out of the Admin Guide).

For now, instead of deleting the old doc pages that had been in the Admin Guide, I removed them from that Admin Guide TOC, fixed up all the "next" links for other docs that previously linked to them, and put a note in the old pages that they've been moved to a new location.

The new enterprise edition guide doesn't yet have "next" links in the footers. Personally I'd rather move to having a fixed sidebar TOC on doc pages for the current guide you're in, but I'm not sure how to do that.

**Comment by mazameli** [comment]
In an effort to get this merged, I've gone ahead and just added Next links to the footers of the enterprise doc pages. I'd like to tackle adding a persistent sidebar TOC to each of our Guides in a separate PR.

Can I ask you to review please, @salsakran?

**Review by salsakran — APPROVED** [review]
the structure works for me

File: docs/users-guide/start.md [hunk 3]
```diff
@@ -17,5 +17,6 @@
 *   [Creating SQL Templates](13-sql-parameters.md)
 *   [Viewing X-ray reports](14-x-rays.md)
 *   [Getting alerts](15-alerts.md)
+*   [Customizing drill-through](16-customizing.drill-through.md)
```
Comment (kdoh) [inline docs/users-guide/start.md]: Should we still be linking to this in the users guide?

Comment (mazameli) [reply docs/users-guide/start.md]: No, good catch!

**Pull request merged**